shopinvader_api_customer/schemas/customer.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from extendable_pydantic import StrictExtendableBaseModel
+
+
+class Customer(StrictExtendableBaseModel, extra="ignore"):
```

Comment:
I think this is a wrong idea to use the same schema for input and output. Typically, we'll want some fields to be required or hidden in one case or another.

Reply:
should be fixed now